addons/mail/static/src/components/composer/composer.scss

```diff
@@ -59,6 +60,11 @@
 .o_Composer_coreFooter {
     grid-area: core-footer;
 
+    &.o-composer-is-compact {
+        max-height: 100px;
+        overflow-x: hidden;
+    }
+
```

Comment:
the overflow should be on attachment list, not whole footer.

Reply:
I moved the max-height but overflow needs to be there otherwise the scrollbar is badly placed due to attachment list margin.
![image](https://user-images.githubusercontent.com/48204996/89627786-a4860b00-d89b-11ea-8673-81c420c89cca.png)

Reply:
should be padding instead of margin

Reply:
Don't change much... Another problem is that we need overflow: hidden on the parent of attachment list to constrain its width without giving "max-width" so that the text ellipsis is actually done on attachment name.

I'm not a CSS expert so I did what I could so that it's working as expected and with your changes I can't manage to make it back working (or I had to add a max-width on attachment which is not that good as in chatter & discuss the width is not the same for example)